presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -447,6 +447,51 @@ Property Name                                        Description
                                                      defaults to ``20``.
 ==================================================== ============================================================
 
+Performance Tuning Configuration Properties
+-------------------------------------------
+
+The following table describes performance tuning properties for the Hive
+connector.
+
+.. warning::
+
+   Performance tuning configuration properties are considered expert-level
+   features. Altering these properties from their default values is likely to
+   cause instability and performance degradation. 
+
+.. list-table:: Hive performance tuning configuration properties
```

Comment:
Please remove the table title, so that it's consistent with other tables. It's also redundant since it's the same as the section header.

Reply:
Done.